changelogs/unreleased/4130-v1tov2-does-not-specify-packages-sufficiently-strict.yml

```
@@ -0,0 +1,6 @@
+description: Limit included namespace packages to inmanta_plugins
```

```suggestion
description: Limit included namespace packages to inmanta_plugins for v1tov2 module conversion.
```
Expanded this a bit because it will be included in the user-visible changelog.